### PR TITLE
Enable automatically generated ad breaks on YouTube

### PR DESCRIPTION
**Pull request by jennygrahamjones**
## What does this change?
In https://github.com/guardian/media-atom-maker/pull/953, we introduced changes to our video uploads by explicitly defining the ad format options applied to each video. In doing this, we may also have been unknowingly applying other options by default. It's not obvious from [YouTube's documentation](https://developers.google.com/youtube/partner/docs/v1/videoAdvertisingOptions#properties) whether these automatic overrides are happening, but users of MAM are reporting that videos which _should_ be eligible for midroll ads - because they are longer than 8 minutes - are not having them applied.

## How to test
In local or CODE, upload a new video clip that is longer than 8 minutes and publish it. Use the `.../api/youtube/video-info/<youtube-id-here>` endpoint in MAM to see which `adBreaks` are applied to the video; these should include midroll. Alternatively/additionally, navigate to the YouTube dashboard for the relevant channel and check the monetisation settings for the video.

## How can we measure success?
Midroll ads are automatically applied to all applicable videos (over 8 minutes long). YouTube channel managers can manage the placement of these ads via the channel's dashboard.

## Images
Screenshot 2020-12-10 at 09 17 49

### CODE REVIEW

**Comment by jennygrahamjones** [comment]
[Midroll ads not enabled on uploads from MAM](https://trello.com/c/5tDAYYZH/2112-midroll-ads-not-enabled-on-uploads-from-mam)

File: docs/08-youtube.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# YouTube
+
+For testing purposes, we have a developer YouTube channel. You can grant other users access to the testing channel in the [Brand Accounts dashboard](https://myaccount.google.com/brandaccounts) on YouTube. If you don't have access, speak to a member of the [Editorial Tools team](https://github.com/orgs/guardian/teams/digital-cms/members).
+
```
Comment (SHession) [inline docs/08-youtube.md:4]:
```suggestion

Troubleshooting: If you have trouble accessing [studio.youtube.com](studio.youtube.com) after having access granted, try navigating to youtube directly and selecting the switch accounts option under your profile picture. 

```

**Review by SHession — APPROVED** [review]
Paired and tested on CODE. LGTM!

**Review by akash1810 — COMMENTED** [review]
Excellent work!

**Pull request merged**